src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1291,9 +1291,16 @@ smart_objects::SmartObjectSPtr MessageHelper::CreateAppVrHelp(
     return NULL;
   }
   smart_objects::SmartObject& vr_help = *result;
-  vr_help[strings::vr_help_title] = app->name();
-
+  vr_help[strings::vr_help_title] = 
+    (*app->vr_help_title())[strings::vr_help_title].asString();
+  
   int32_t index = 0;
+
+  smart_objects::SmartObject so_vr_help(smart_objects::SmartType_Map);
+  so_vr_help[strings::position] = index+1;
```

Comment:
@ValeriiMalkov please run `check_style.sh` as I could see some style errors

Reply:
@AKalinich-Luxoft That was fixed at the commit 8b18082df6387812f7a92870da98a12c9db9b15e